lib/modules/manager/composer/readme.md

```diff
@@ -1,3 +1,10 @@
+<!-- prettier-ignore -->
+!!! warning
+    Renovate does not support Composer version `2.5.0` or newer.
+    This is because of changes in the default behavior of Composer.
+    For now, you can do ... to workaround the problem.
+    Subscribe to these issues/discussions to follow our progress: insert link to issues.
```

Comment:
Do we want to point to our discussion or upstream issue? We should prevent people piling on the maintainers of Composer, so maybe best to only link to Renovate's issues?